penney/functions.py

```diff
@@ -131,11 +131,11 @@ def C_calc(seq_dict):
     C = []
     names = sorted(list(seq_dict.keys()))
     p_seq = lambda seq: 1 / 2 ** len(seq)
-    for i in range(len(names)):
-        A_i = seq_dict[str(names[i])]
+    for name in names:
+        A_i = seq_dict[str(name)]
         C_row = []
-        for j in range(len(names)):
-            A_j = seq_dict[str(names[j])]
+        for name in names:
```

Comment:
I think, using of same `iter` name in a nested loop is not a good approach ;-)
I suggest to use two different names, like `name1` and `name2` ...

Reply:
You are completely right.